### PR TITLE
Add markers to the map

### PR DESCRIPTION
**Pull request by KathleenQ**
The added markers are with personalised marker icons.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
   </head>
   <body onload=fetchBlobstoreUrlAndShowForm()>
-    <div id="content">
+    <div id="content" style="width: 750px">
```
Comment (aannie-y) [inline portfolio/src/main/webapp/index.html:11]: maybe put the styling in the style sheet?

**Review by aannie-y — COMMENTED** [review]
lgtm

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (KathleenQ) [reply portfolio/src/main/webapp/index.html:11]: Yes. Thank you to point it out! I have fixed it in PR #10 .

**Comment by lilian-hunt** [comment]
lgtm

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**